packages/manager/modules/overthebox/src/overthebox/autoconfigure/dhcp/template.html

```diff
@@ -0,0 +1,284 @@
+<div>
+    <oui-message
+        data-ng-if="$ctrl.successMessage"
+        type="success"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        on-dismiss="$ctrl.onDismiss()"
+        dismissable
+    >
+        <span data-ng-bind="$ctrl.successMessage"></span>
+    </oui-message>
+    <oui-message
+        data-ng-if="$ctrl.errorMessage"
+        type="error"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        on-dismiss="$ctrl.onDismiss()"
+        dismissable
+    >
+        <span data-ng-bind="$ctrl.errorMessage"></span>
+    </oui-message>
+    <div class="oui-box oui-box_light">
+        <h4
+            class="oui-box__heading"
+            data-translate="overTheBox_autoconfigure_dhcp_config"
+        ></h4>
+        <div data-ng-if="$ctrl.configList.length === 0" class="mb-2">
+            <em data-translate="overTheBox_autoconfigure_no_values"></em>
+        </div>
+        <div data-ng-if="$ctrl.showAddConfig">
+            <form novalidate name="formAddConf">
+                <div class="oui-input-group oui-input-group_inline">
+                    <oui-field
+                        label="{{:: 'overTheBox_autoconfigure_dhcp_interface' | translate}}"
+                        label-popover="{{:: 'overTheBox_autoconfigure_dhcp_interface_description' | translate}}"
+                    >
+                        <input
+                            class="oui-input"
+                            name="interface"
+                            type="text"
+                            ng-model="$ctrl.config.interface"
+                        />
+                    </oui-field>
+                    <oui-field
+                        class="ml-2"
+                        label="{{:: 'overTheBox_autoconfigure_dhcp_lease_duration' | translate}} *"
```

Comment:
**nitpick:**  Please avoid using `*` to mark field as required. Prefer using `(optional)` suffix wherever is needed.

```suggestion
                        label="{{:: 'overTheBox_autoconfigure_dhcp_lease_duration' | translate}}"
```